.rubocop.yml

```diff
@@ -121,3 +122,21 @@ Style/TrailingWhitespace:
 Style/VariableName:
   Severity: fatal
   Enabled: true
+
+Metrics/LineLength:
+  Description: 'Limit lines to 120 characters.'
```

Comment:
I think this setting is more about code design. 

Anyone writing a line of code longer than let's say 100 characters have probably got something smelly going on (not always, but it's definitely an indicator)